packages/devtools_app/lib/src/performance/performance_controller.dart

```diff
@@ -583,45 +586,49 @@ class PerformanceController extends DisposableController
     for (TraceEvent event in threadNameEvents) {
       final name = event.args['name'];
 
-      // Android: "1.ui (12652)"
-      // iOS: "io.flutter.1.ui (12652)"
-      // MacOS, Linux, Windows, Dream (g3): "io.flutter.ui (225695)"
-      if (name.contains('.ui')) {
-        uiThreadId = event.threadId;
-      }
+      if (isInitialUpdate) {
```

Comment:
we should also check if we are connected to a flutter app (either offline or online). See an example of the below check here: https://github.com/flutter/devtools/blob/a19d96fc321d047f5b96d3b1d7d5a8a32cdf3eb6/packages/devtools_app/lib/src/performance/performance_screen.dart#L151-L153

```
final isOfflineFlutterApp = offlineController.offlineMode.value &&
       offlinePerformanceData != null &&
        offlinePerformanceData.frames.isNotEmpty;

if (isInitialUpdate && (isOfflineFlutterApp ||
            (!offlineController.offlineMode.value &&
                serviceManager.connectedApp.isFlutterAppNow))
```

Reply:
Done